lua/nvim-treesitter/parsers.lua

```diff
@@ -289,13 +289,11 @@ list.cue = {
 
 list.d = {
   install_info = {
-    url = "https://github.com/CyberShadow/tree-sitter-d",
-    files = { "src/parser.c", "src/scanner.cc" },
-    requires_generate_from_grammar = true,
+    url = "https://github.com/gdamore/tree-sitter-d",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
```

Comment:
If `main` is the default branch, this can be omitted.

Reply:
I had some issues loading it on my nvim without it

Reply:
That doesn't sound like the actual issue; the problem was more likely (since it usually is) the stale lockfile commit hash.

Try dropping it.